src/libra_py/dynamics/qtag/qtag_plots.py

```diff
@@ -0,0 +1,270 @@
+"""
+..module:: qtag_plots
+  :platform: Unix, Windows
+  :synopsis: This module contains functions for plotting various qtag outputs.
+
+..moduleauthors :: Matthew Dutra
+"""
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import cm
+
+from libra_py import data_read
+
+def _plot_wf_1D(plt_params, data, *ref_data):
```

Comment:
why do you start the function names with the underscore? please remove here and below since this is not the best practice